enigma-principal/app/src/epoch_u/epoch_provider.rs

```diff
@@ -223,18 +267,29 @@ impl EpochProvider {
 //////////////////////// TESTS  /////////////////////////////////////////
 
 #[cfg(test)]
-mod test {
-    use web3::types::{Bytes, H160};
+pub mod test {
     use std::collections::HashMap;
+
+    use web3::types::{Bytes, H160};
+
+    use enigma_crypto::KeyPair;
+    use enigma_tools_u::{esgx::general::storage_dir};
     use enigma_types::ContractAddress;
 
     use super::*;
 
     pub const WORKER_SIGN_ADDRESS: [u8; 20] =
         [95, 53, 26, 193, 96, 206, 55, 206, 15, 120, 191, 101, 13, 44, 28, 237, 80, 151, 54, 182];
 
+    pub fn setup_epoch_storage() {
+        let mut path = storage_dir(ENCLAVE_DIR).unwrap();
+        path.push(EPOCH_DIR);
+        fs::remove_dir_all(path).unwrap_or_else(|_| println!("Epoch dir already empty"));
```

Comment:
FYI you have `expect(msg)` for this

Reply:
Doesn't `expect` panic just like `unwrap`. I want it do continue of the directory is already removed (possibly by another test running concurrently).

Reply:
Oh. ok